src/main/target/COLIBRI/config.c

```diff
@@ -33,6 +33,6 @@ void targetConfiguration(void)
 {
     mixerConfigMutable()->mixerMode = MIXER_HEX6X;
     rxConfigMutable()->serialrx_provider = 2;
```

Comment:
Would be better to use the constant in the `SerialRxType` enum rather than a magic `2`.